extensions/ql-vscode/src/remote-queries/repository-selection.ts

```diff
@@ -30,7 +33,33 @@ interface RepoList {
  * Gets the repositories or repository lists to run the query against.
  * @returns The user selection.
  */
-export async function getRepositorySelection(): Promise<RepositorySelection> {
+export async function getRepositorySelection(
+  dbManager?: DbManager,
+): Promise<RepositorySelection> {
+  if (isNewQueryRunExperienceEnabled()) {
+    const selectedDbItem = await dbManager?.getSelectedDbItem();
```

Comment:
Don't need to await here - the function isn't asynchronous.

```suggestion
    const selectedDbItem = dbManager?.getSelectedDbItem();
```

Reply:
👍